blog/views.py

```diff
@@ -0,0 +1,50 @@
+from django.urls import reverse_lazy
+from django.views.generic import (
+    ListView, DetailView, CreateView, UpdateView, DeleteView
+)
+from django.core.mail import mail_admins
+from .models import Post
+from .forms import PostForm
+
+class PostListView(ListView):
+    model = Post
```

Comment:
Правильнее было либо указать `queryset`, вместо `model` и метода `get_queryset`
```suggestion
    queryset = Post.objects.filter(is_published=True)
```
Либо все таки оставить `model`, тогда в `get_queryset` использовать наследование:
```python
def get_queryset(self):
    qs = super().get_queryset()
    return qs.filter(is_published=True)
```